fastlane/lib/fastlane/actions/git_add.rb

```diff
@@ -7,9 +7,9 @@ def self.run(params)
           UI.success("Successfully added from \"#{paths}\" 💾.")
         elsif params[:path]
```

Comment:
Hey @thii sorry for the back and forth... If I'm understanding the purpose of this PR correctly, shouldn't we be combining the pathspec and the path params to add all the files specified by those params instead of checking with `elsif` here?

Let me know if I am just misunderstanding the PR! 🐙